amundsen_application/static/css/_layouts.scss

```diff
@@ -94,11 +96,11 @@ $resource-header-height: 84px;
       flex-basis: 600px;
       flex-shrink: 0;
       overflow-y: scroll;
-      padding: 0 24px;
+      padding: $aside-separation-space 24px;
```

Comment:
We now have double the amount of space between the header and the first section in the left-panel. So I think we'd want to remove this padding here and on line 159.

Reply:
Are you sure? 
I am comparing it with Staging and looks the same to me!

Reply:
Got it, fixing